core/templates/dev/head/domain/statistics/LearnerAnswerDetailsBackendApiServiceSpec.ts

```diff
@@ -54,7 +54,7 @@ describe('Learner answer info backend Api service', function() {
 
       $httpBackend.expect(
         'PUT',
-        '/explorehandler/learner_answer_details/exploration/exp123').respond(
```

Comment:
Shouldn't there be a corresponding backend change for this?

Reply:
https://github.com/oppia/oppia/blob/cfc8d7543483327ff71a62c278ef78e5d052ccdc/feconf.py#L651

Here its done